mat/lq.go

```diff
@@ -40,12 +40,12 @@ func (lq *LQ) updateCond(norm lapack.MatrixNorm) {
 	putInts(iwork)
 }
 
-// Factorize computes the LQ factorization of an m×n matrix a where n <= m. The LQ
+// Factorize computes the LQ factorization of an m×n matrix a where m <= n. The LQ
 // factorization always exists even if A is singular.
 //
 // The LQ decomposition is a factorization of the matrix A such that A = L * Q.
 // The matrix Q is an orthonormal n×n matrix, and L is an m×n upper triangular matrix.
```

Comment:
L is _lower_ triangular. Can you please fix also this?

Reply:
Done. Now we know for sure that this was a copy paste error.